packages/playwright-test/src/runner/dispatcher.ts

```diff
@@ -131,7 +132,15 @@ export class Dispatcher {
       worker = this._createWorker(job, index, serializeConfig(this._config));
       this._workerSlots[index].worker = worker;
       worker.on('exit', () => this._workerSlots[index].worker = undefined);
-      await worker.start();
+      try {
+        await worker.start();
+      } catch (error) {
+        // When failed to start after stopping, most likely not-yet-intialized subprocess
+        // got SIGINT'ed. It is safe to ignore.
+        if (!this._isStopped)
+          this._markAllTestsInJobAsFailedOrSkipped(job, error);
```

Comment:
It feels wrong, i.e. looks like it should be handled the same way as a crashed worker.